docs/Customization/Memory-Hierarchy.rst

```diff
@@ -1,4 +1,104 @@
 Memory Hierarchy
 ===============================
-TODO: Talk about SiFive Cache, and integration with L1 and backing main memory models
-(maybe even Tilelink)
+
+The L1 Caches
+--------------
+
+Each CPU tile has an L1 instruction cache and L1 data cache. The size and
+associativity of these caches can be configured. The default ``RocketConfig``
+uses 16 KiB, 4-way set-associative instruction and data caches. However,
+if you use the ``NMediumCores`` or ``NSmallCores`` configurations, you can
```

Comment:
Do we want to mention the specific parameters you can change to change to L1 cache sizes to something custom? (i.e., not the NMediumCores, but rather DCacheParams)

Reply:
But the only way to get at those parameters is to override RocketTilesKey. I feel like if they want to do something tricky like that, they can just look at the code for `NMediumCores` and `NSmallCores`.

Reply:
Then maybe it's worth adding a sentence saying "for custom cache sizes, associativity, etc., you will need to modify the RocketTilesKey parameters, as is specified in the the Parameter System <link> section (and then we should add something about this in the parameter system section).

Reply:
But that section talks about the config system more generally and not about specific keys. Though we could add something about how to use `site` and `up` to reference other keys.

Reply:
We could and we should